compiler/src/iree/compiler/DispatchCreation/WrapEncodingOpInDispatchRegion.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright 2025 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree/compiler/Dialect/Encoding/IR/EncodingDialect.h"
+#include "iree/compiler/Dialect/Encoding/IR/EncodingOps.h"
+#include "iree/compiler/Dialect/Flow/IR/FlowDialect.h"
+#include "iree/compiler/Dialect/Flow/IR/FlowOps.h"
+#include "iree/compiler/Dialect/Flow/Transforms/RegionOpUtils.h"
+#include "iree/compiler/DispatchCreation/Passes.h"
+namespace mlir::iree_compiler::DispatchCreation {
```

Comment:
style nit: add a blank line before namespace.

```suggestion
#include "iree/compiler/DispatchCreation/Passes.h"

namespace mlir::iree_compiler::DispatchCreation {
```